velox/exec/Task.h

```diff
@@ -194,7 +194,8 @@ class Task : public std::enable_shared_from_this<Task> {
   /// of buffers. No more calls are expected after the call with noMoreBuffers
   /// == true, but occasionally the caller might resend it, so calls
   /// received after a call with noMoreBuffers == true are ignored.
-  void updateBroadcastOutputBuffers(int numBuffers, bool noMoreBuffers);
+  /// @return bool Return true if the update was successful, else false.
```

Comment:
Maybe clarify why the update may fail, e.g. the task execution fails or this calls come after a call with noMoreBuffers == true.